sentry_sdk/integrations/falcon.py

```diff
@@ -28,6 +28,21 @@
     raise DidNotEnable("Falcon not installed")
 
 
+# In Falcon 3.0 `falcon.api_helpers` is renamed to `falcon.app_helpers`
+# and `falcon.API` to `falcon.App`
+try:
+    import falcon.app  # type: ignore
+    import falcon.app_helpers  # type: ignore
```

Comment:
It would be preferable to attempt to import `falcon.app_helpers` first, then import `falcon.api_helpers` if the first fails.
Otherwise, the current code will keep complaining with deprecation warnings that api_helpers has been renamed.